docs/commands/ft._list.md

```
@@ -16,4 +18,10 @@ ft._list
 2) products
 3) users
 4) transactions
-```
\ No newline at end of file
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)